gradle/libs.versions.toml

```diff
@@ -20,3 +20,4 @@ junit-launcher = { module = "org.junit.platform:junit-platform-launcher" }
 antlr = { module = "org.antlr:antlr4", version.ref = "antlr" }
```

Comment:
Do we need this file into the project? Or is it just about your IDE?

Reply:
we need it to generate the parser files